mmdet/datasets/loader/sampler.py

```diff
@@ -4,7 +4,6 @@
 import torch
 import numpy as np
 
-from torch.distributed import get_world_size, get_rank
```

Comment:
You may use `mmcv.runner.get_dist_info()`(https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/utils.py#L16) instead of `torch.distributed.get_xxx()`, which handles the exceptions properly.

Reply:
Got point, thanks!